packages/sagas/package.json

```diff
@@ -56,6 +56,7 @@
     "redux-saga": "0.15.4"
   },
   "devDependencies": {
+    "babel": "^6.23.0",
```

Comment:
Why was babel added?

Reply:
because without it local package build can't be achieved